Tests/TuistGeneratorTests/GraphViz/GraphToGraphVizMapperTests.swift

```diff
@@ -39,25 +39,13 @@ final class GraphToDotGraphMapperTests: XCTestCase {
 
         // When
         let got = subject.map(graph: graph, skipTestTargets: false, skipExternalDependencies: false)
-
-        // Then
-        let expected = DotGraph(name: "Project Dependencies Graph",
-                                type: .directed,
-                                nodes: Set([
-                                    .init(name: "Tuist iOS"),
-                                    .init(name: "CoreData"),
-                                    .init(name: "RxSwift"),
-                                    .init(name: "XcodeProj"),
-                                    .init(name: "Core"),
-                                    .init(name: "Tuist watchOS"),
-                                ]), dependencies: [
-                                    .init(from: "Tuist iOS", to: "Core"),
-                                    .init(from: "Tuist watchOS", to: "Core"),
-                                    .init(from: "Core", to: "XcodeProj"),
-                                    .init(from: "Core", to: "RxSwift"),
-                                    .init(from: "Core", to: "CoreData"),
-                                ])
-        XCTAssertEqual(got, expected)
+        let expected = makeExpectedGraphViz()
```

Comment:
Looks like `Graph` is equatable, may help simplify the equality check here?

Reply:
I'm not sure about this one, it's kind of a pain to work with `GraphViz.Graph` equality, since the model has a lot of properties. For example, when equality failed, I had a 12k characters message to tell me what was the difference, that I couldn't spot.
I'd keep it easy and use node ids, unless we want to bring in something like https://github.com/krzysztofzablocki/Sourcery/pull/68/files cc @pepibumur